tools/ci/travis-install.py

```diff
@@ -48,23 +47,26 @@ def install(omp=False, mpi=False, phdf5=False):
     else:
         cmake_cmd.append('-DHDF5_PREFER_PARALLEL=OFF')
 
+    if dagmc:
+        cmake_cmd.append('-Ddagmc=ON')
+        
     # Build and install
     cmake_cmd.append('..')
     print(' '.join(cmake_cmd))
     subprocess.check_call(cmake_cmd)
-    subprocess.check_call(['make', '-j'])
+    subprocess.check_call(['make', '-j4'])
```

Comment:
Just to be clear, do I understand correctly that you'll get memory errors if you just use `-j` rather than `-j4` here?

Reply:
That's correct. It's not clear why at this point.